server/src/main/java/org/candlepin/dto/api/v1/ProductCertificateDTO.java

```diff
@@ -0,0 +1,196 @@
+ *
+ * Red Hat trademarks are not licensed under GPLv2. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+package org.candlepin.dto.api.v1;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.annotations.ApiModel;
+import org.apache.commons.lang.builder.EqualsBuilder;
+import org.apache.commons.lang.builder.HashCodeBuilder;
+import org.candlepin.common.jackson.HateoasInclude;
+
+/**
+ * The ProductCertificateDTO is a DTO representing product certificates presented to the API.
+ */
+@ApiModel(parent = TimestampedCandlepinDTO.class, description = "DTO representing a product certificate")
+public class ProductCertificateDTO extends TimestampedCandlepinDTO<ProductCertificateDTO> {
+
+    private static final long serialVersionUID = 1L;
+
+    protected String id;
+    protected String key;
+    protected String cert;
+    private ProductDTO product;
+
+    /**
+     * Initializes a new ProductCertificateDTO instance with null values.
+     */
+    public ProductCertificateDTO() {
+        // Intentionally left empty
+    }
+
+    /**
+     * Initializes a new ProductCertificateDTO instance which is a shallow copy of the provided
+     * source entity.
+     *
+     * @param source
+     *  The source entity to copy
+     */
+    public ProductCertificateDTO(ProductCertificateDTO source) {
+        super(source);
+    }
+
+    /**
+     * Retrieves the id field of this ProductCertificateDTO object.
+     *
+     * @return the id field of this ProductCertificateDTO object.
+     */
+    @HateoasInclude
+    public String getId() {
+        return this.id;
+    }
+
+    /**
+     * Sets the id to set on this ProductCertificateDTO object.
+     *
+     * @param id the id to set on this ProductCertificateDTO object.
+     *
+     * @return a reference to this ProductCertificateDTO object.
+     */
+    public ProductCertificateDTO setId(String id) {
+        this.id = id;
+        return this;
+    }
+
+    /**
+     * Returns the product of this product certificate.
+     *
+     * @return the owner of this entitlement.
+     */
+    @JsonIgnore
+    public ProductDTO getProduct() {
+        return this.product;
+    }
+
+    /**
+     * Sets the owner of this entitlement.
+     *
+     * @param product the product to set.
+     *
+     * @return a reference to this ProductCertificateDTO object.
+     */
+    @JsonProperty
+    public ProductCertificateDTO setProduct(ProductDTO product) {
+        this.product = product;
+        return this;
+    }
+
+    public String getKey() {
+        return this.key;
+    }
+
+    public ProductCertificateDTO setKey(String key) {
+        this.key = key;
+        return this;
+    }
+
+    public String getCert() {
+        return this.cert;
+    }
+
+    public ProductCertificateDTO setCert(String cert) {
```

Comment:
Minor suggestion (feel free to ignore): these setters/getters should also have comments just for consistency.